src/Course/List.hs

```diff
@@ -291,7 +372,14 @@ question_List_7 = error "TODO"
 --   (e) What is the type of `mapOptional map`? What does it do?
 --   (f) What is the type of `map mapOptional`? What does it do?
 question_List_8 :: (String, String, String, String, String, String)
-question_List_8 = error "TODO"
+question_List_8 =
+  ( "`map . map` is of type `(a -> b) -> List (List a) -> List (List b)` and instead maps `f` onto a list of lists."
+  , "`map map` is of type `List (a -> b) -> List (List a -> List b)` which takes a list of functions and returns a list of lifted functions."
+  , "`map . mapOptional` is of type `(a -> b) -> List Optional a -> List Optional b` which takes a function, lifts it into the `Optional` type, then lifts it into the `List` type to let you map a function over a list of optional values."
```

Comment:
```suggestion
  , "`map . mapOptional` is of type `(a -> b) -> List (Optional a) -> List (Optional b)` which takes a function, lifts it into the `Optional` type, then lifts it into the `List` type to let you map a function over a list of optional values."
```

Without the parentheses, you're passing _two_ type arguments to `List`, but `List` expects only one type argument.